src/components/create-job-form/index.js

```diff
@@ -18,13 +18,15 @@ class CreateJobForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      pipelineTypes: ['build', 'build-deploy'],
```

Comment:
This shouldn't be in the component `state`; just make it a `const` at the top of the file?